fastlane/lib/fastlane/fast_file.rb

```diff
@@ -1,3 +1,26 @@
+require "rubygems"
```

Comment:
Is this gem installed on every system?

Reply:
A few thoughts.

1. This is actually not a gem, but part of the Ruby library. And you always get RubyGems when you install Ruby. (You just don't get the Bundler until you install it _from_ RubyGems.) In an RVM-based install, it's located here, e.g.:

```bash
-rw-r--r--  1 jdee  staff  36192 Sep  9 22:23 /Users/jdee/.rvm/rubies/ruby-2.4.2/lib/ruby/2.4.0/rubygems.rb
```

2. This `require` may not be necessary.

```bash
[jdee@Jimmy-Dees-MacBookPro settings-bundle (master)]$ irb
2.4.2 :001 > Gem::Version.new "0.1.0" # no require
 => #<Gem::Version "0.1.0"> 
```

3. It seems safest to require this, given general require-related concerns (cc: @milch). But it probably doesn't make much difference and could potentially be omitted.

Reply:
Sounds good to me 👍

Reply:
Also `require "rubygems"` already occurs in a number of source files:

```
[jdee@Jimmy-Dees-MacBookPro fastlane (wakka-wakka)]$ find . -type f -name '*.rb' -exec grep -l 'require.*rubygems' {} \;
./fastlane_core/lib/fastlane_core/tag_version.rb
./fastlane/spec/plugins_specs/plugin_generator_spec.rb
./fastlane/lib/fastlane/fast_file.rb
./fastlane/lib/fastlane/fastlane_require.rb
./fastlane/lib/fastlane/actions/update_fastlane.rb
./fastlane/lib/fastlane/actions/xcversion.rb
```

Reply:
What is it actually used for? Only the `Gem::Version` class? If so it might be possible to do a more specific `require`, e.g. `require  'rubygems/version'` or something like that

Reply:
Yeah, I had that thought. `require "rubygems/requirement"` in fast_file.rb and `require "rubygems/version"` in tag_version.rb.